### ISSUE TITLE
Opening hours category

### ISSUE THREAD

**Issue by ronan-mch** [ronan-mch/det_sker#243]
Show on Location view

**Comment by ronan-mch** [comment]
not sure if this is the right approach - might be better doing it as a property of the location.

**Comment by Siggik** [comment]
Ja, tror det sidste er en god idé. Det kunne godt være en ting, vi som admins skulle påtage os at holde opdateret?
